pipelines/institution_id/00-data-assessment-TEMPLATE.py

```diff
@@ -590,6 +612,102 @@
 
 # COMMAND ----------
 
+# First year GPA by cohort and enrollment type
+# to remove error bars, errorbar=None
+ax = sb.barplot(
+    df_cohort_valid.sort_values(by="cohort").astype(
+        {"gpa_group_year_1": "Float32"}
+    ),
+    x="cohort",
+    y="gpa_group_year_1",
+    estimator="mean",
+    hue="enrollment_type",
+    edgecolor="white",
+)
+
+# Set the ylabel
+ax.set(ylabel="Avg. GPA (Year 1)")
+
+# Move the legend to a different location (e.g., upper left)
+ax.legend(loc="lower left", title="Enrollment Type")
+
+# Show the plot
+plt.show()
```

Comment:
no need for code comments that simply describe the code being executed :)

```suggestion
ax.set(ylabel="Avg. GPA (Year 1)")
ax.legend(loc="lower left", title="Enrollment Type")
```